src/schema/rules/files/raw/photo.yaml

```diff
@@ -0,0 +1,30 @@
+---
+photo:
+  suffixes:
+    - photo
+  extensions:
+    - .jpg
+    - .png
+    - .tif
+  datatypes:
+    - eeg
+    - ieeg
+    - meg
+    - nirs
+  entities:
+    subject: required
+    session: optional
+    acquisition: optional
```

Comment:
@rob-luke @sappelhoff I didn't follow the discussion of `photo.*` closely. Was there a reason that nirs photos were missing the optional acquisition flag? I can break this back out into a separate rule, but I wanted to check that this wasn't an oversight first.

Reply:
Actually, now I'm seeing that the rule was never rendered, and there's no mention in the text. Should I just remove it?